### DISCUSSION TITLE
Yale Linus

### DISCUSSION THREAD

**Discussion by Marek-SSS** [homebridge-plugins/homebridge-august#40] [General]
Hello,
do you plan to support Yale Linus WiFi bridge?

Marek

**Comment by donavanbecker** [comment]
Have you tried to connect it?

**Comment by Marek-SSS** [comment]
﻿Yes, homebridge and Yale-wifibridge under the same Wi-Fi network. I was registered August account (for homebrige-August) using email address but from Yale apps I can give permission only for mobile numbers (no emails). I think this could be a problem.

Log:
[12/17/2022, 5:06:05 PM] [August] Discover Devices: TypeError: Cannot convert undefined or null to object
